include/macros.h

```diff
@@ -12,7 +12,7 @@
 
 #define SQ(x) ((x)*(x))
 #define ABS(x) ((x) >= 0 ? (x) : -(x))
-#define	ULTRA_ABS(x) ((x) > 0) ? (x) : -(x)
+#define ULTRA_ABS(x) ((x) > 0) ? (x) : -(x)
```

Comment:
When this was added I was pretty sure roman said this would fit better in ultra related headers or even the specific file that its used in. I could be wrong but @Roman971 can you confirm

Reply:
Yeah I still think so